test-automation-framework/org.wso2.carbon.automation.extensions/src/main/java/org/wso2/carbon/automation/extensions/ExtensionConstants.java

```diff
@@ -54,4 +60,18 @@
     public static final String HTML_UNIT_DRIVER = "htmlUnit";
     public static final String OPERA_BROWSER = "opera";
     public static final String CARBON_HOME = "carbon.home";
+    public static final String TEST_GROUP_PARAMETER = "testGroup";
+    public static final String CONF_LOCATION_PARAMETER = "confLocation";
+    public static final String CONF_LOCATION = "conf.location";
+    public static final String TOMCAT_PORT_PARAMETER = "tomcatPort";
+    public static final String TOMCAT_APPLICATIONS_LOCATIONS_PARAMETER = "tomcatWebAppsLocations";
+    public static final String TOMCAT_APPLICATIONS_URLS_PARAMETER = "tomcatWebAppsUrls";
+    public static final String PARAMETER_SEPARATOR = ",";
+    public static final int DEFAULT_TOMCAT_PORT = 8049;
+    public static final String ADDITIONAL_IS_COUNT_PARAMETER = "additionalISCount";
+    public static final String ADDITIONAL_IS_PORT_OFFSETS_PARAMETER = "additionalSPortOffsets";
+    public static final String ADDITIONAL_IS_PROUDUCT_GROUP_NAMES_PARAMETER = "additionalISProductGroupNames";
+    public static final String ADDITIONAL_IS_INSTANCE_NAMES_PARAMETER = "additionalISInstanceNames";
+    public static final String ADDITIONAL_IS_TEST_USER_MODES_PARAMETER = "additionalISTestUserModes";
+    public static final String ADDITIONAL_IS_PARAMETER_VALUE_SEPARATOR = ",";
```

Comment:
Is there a particular reason why IS specific constants belong in automation framework? These are not used within this pr as I see.